Wikipedia/Code/ExploreViewController.swift

```diff
@@ -1319,13 +1383,16 @@ extension ExploreViewController {
 
     @objc func applicationDidBecomeActive() {
         if !UIAccessibility.isVoiceOverRunning {
+            presentYearInReviewAnnouncement()
```

Comment:
I worry about this line and the next interacting with each other, or causing two feature announcement presentations back-to-back for different features. If you move a boolean method into the data controller maybe you could do something here like:

```
if yirDataController.shouldShowYearInReviewFeatureAnnouncement() {
   presentYearInReviewAnnouncement()
} else {
   presentImageRecommendationsFeatureAnnouncementIfNeeded
}
```

Also note that we probably want this one to show if VoiceOver is running.

This call presents it after foregrounding, but Explore's `viewDidAppear` also calls `presentImageRecommendationsFeatureAnnouncementIfNeeded`. Do we need `presentYearInReviewAnnouncement` there as well?